deployment/azuredeploy.json

```diff
@@ -446,54 +413,6 @@
                 ]
             }
         },
-        {
-            "type": "Microsoft.Resources/deploymentScripts",
-            "apiVersion": "2019-10-01-preview",
-            "name": "[variables('cliResourceName')]",
-            "location": "[resourceGroup().location]",
-            "kind": "AzureCLI",
-            "dependsOn": [
-                "[resourceId('Microsoft.KeyVault/vaults', variables('keyVaultName'))]",
-                "[resourceId('Microsoft.Storage/storageAccounts', variables('staticWebsite'))]",
-                "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities',parameters('identityName'))]"
-            ],
-            "identity": {
-                "type": "UserAssigned",
-                "userAssignedIdentities": {
-                    "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities',parameters('identityName'))]": {
-                    }
-                }
-            },
-            "properties": {
-                "forceUpdateTag": "[parameters('utcValue')]",
-                "AzCliVersion": "2.0.80",
-                "timeout": "PT30M",
-                "arguments": "[variables('cliArg')]",
-                "scriptContent": "az account list",
-                "cleanupPreference": "OnSuccess",
-                "retentionInterval": "P1D"
-            }
-        },
-        {
-            "type": "Microsoft.ManagedIdentity/userAssignedIdentities",

```

Comment:
No dependency of any stage on `userAssignedIdentities` 🤔?

Reply:
No, it was only created to give the Managed Identity access to the key vault